requirements_tests.txt

```diff
@@ -12,3 +12,4 @@ pytest==3.0.7
 six==1.10.0
 SQLAlchemy==1.1.6
 Werkzeug==0.12.1
+-e git+ssh://git@github.com/closeio/zbase62.git@41253f397d552cf75476a85d345a8490601d8f40#egg=zbase62
```

Comment:
Use `e13d2c748ccdb0cafe6465961a0c6a4111ee219f` (https://github.com/closeio/zbase62/tree/python3-fixes)

Reply:
Nice, that worked